ledger/src/blockstore_db.rs

```diff
@@ -337,10 +337,14 @@ impl Column for columns::TransactionStatus {
     }
 
     fn index(key: &[u8]) -> (u64, Signature, Slot) {
-        let index = BigEndian::read_u64(&key[0..8]);
-        let signature = Signature::new(&key[8..72]);
-        let slot = BigEndian::read_u64(&key[72..80]);
-        (index, signature, slot)
+        if key.len() != 80 {
```

Comment:
Can we also delete https://github.com/solana-labs/solana/blob/master/ledger/src/blockstore_db.rs#L673-L679, since the guarantees described there no longer hold for all columns?

Reply:
Yes, I think so. As discussed, we don't seem to be using that result anywhere anyway.